jitbuilder/optimizer/JBOptimizer.cpp

```diff
@@ -149,22 +152,61 @@ Optimizer::Optimizer(TR::Compilation *comp, TR::ResolvedMethodSymbol *methodSymb
+   _opts[OMR::basicBlockOrdering] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_OrderBlocks::create, OMR::basicBlockOrdering);
+   _opts[OMR::globalCopyPropagation] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_CopyPropagation::create, OMR::globalCopyPropagation);
+   _opts[OMR::globalDeadStoreElimination] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_DeadStoreElimination::create, OMR::globalDeadStoreElimination);
+   _opts[OMR::basicBlockHoisting] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_HoistBlocks::create, OMR::basicBlockHoisting);
+   _opts[OMR::globalValuePropagation] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR::GlobalValuePropagation::create, OMR::globalValuePropagation);
+   _opts[OMR::localValuePropagation] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR::LocalValuePropagation::create, OMR::localValuePropagation);
+   _opts[OMR::trivialDeadTreeRemoval] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_TrivialDeadTreeRemoval::create, OMR::trivialDeadTreeRemoval);
+   _opts[OMR::generalLoopUnroller] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_GeneralLoopUnroller::create, OMR::generalLoopUnroller);
+   _opts[OMR::basicBlockExtension] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_ExtendBasicBlocks::create, OMR::basicBlockExtension);
+   _opts[OMR::redundantGotoElimination] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_EliminateRedundantGotos::create, OMR::redundantGotoElimination);
+   _opts[OMR::rematerialization] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_Rematerialization::create, OMR::rematerialization);
+   _opts[OMR::loopCanonicalization] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_LoopCanonicalizer::create, OMR::loopCanonicalization);
+   _opts[OMR::inductionVariableAnalysis] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_InductionVariableAnalysis::create, OMR::inductionVariableAnalysis);
+   _opts[OMR::liveRangeSplitter] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_LiveRangeSplitter::create, OMR::liveRangeSplitter);
+   _opts[OMR::tacticalGlobalRegisterAllocator] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_GlobalRegisterAllocator::create, OMR::tacticalGlobalRegisterAllocator);
+   _opts[OMR::regDepCopyRemoval] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR::RegDepCopyRemoval::create, OMR::regDepCopyRemoval);
+   _opts[OMR::inlining] =
+      new (comp->allocator()) TR::OptimizationManager(self(), TR_TrivialInliner::create, OMR::inlining);
+
+   // Initialize optimization groups
+//   _opts[OMR::isolatedStoreGroup] =
+//      new (comp->allocator()) TR::OptimizationManager(self(), NULL, OMR::isolatedStoreGroup, isolatedStoreOpts);
```

Comment:
Any reason why this group is commented out already and not simply deleted?  If you need to leave it commented out please leave a comment explaining why.  :-)

Reply:
i removed them, i don't see why they need to be there